### ISSUE TITLE
[Enhancement]: Product Category Display Accessibility

### ISSUE THREAD

**Issue by MarkHollo** [woothemes/woocommerce#34109]
### Describe the solution you'd like

We've been receiving feedback from clients as their websites are mandatory to pass the accessibility tests in full (Europe), so we're getting back to you in regards to reported issues along with proposals on addressing these. We didn't create pull requests as we figured you might want to use a different approach in this particular matter.

Product category displays to pass accessibility tests. For more information, check the [Wave tool](https://wave.webaim.org). 

Affected line:
https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/includes/wc-template-functions.php#L1220

### Describe alternatives you've considered

N/A

### Additional context

Proposed solution:

function woocommerce_template_loop_category_link_open( $category ) {
	 	// Vars.
		$term        = get_term( $category->term_id, 'product_cat' );
		$cat_name    = $term->name;
		
		echo '<a aria-label="' . esc_attr__( 'Visit product category', 'woocommerce' ) . ' ' . $cat_name . '" href="' . esc_url( get_term_link( $category, 'product_cat' ) ) . '">';
	}

**Comment by woocommercebot** [comment]
Hi @MarkHollo,

  Thank you for opening the issue! It requires further feedback from the WooCommerce Core team.

  We are adding the `needs developer feedback` label to this issue so that the Core team could take a look.

  Please note it may take a few days for them to get to this issue. Thank you for your patience.

**Comment by peterfabian** [comment]
Hi @MarkHollo

Apologies for taking a long time to respond! Your suggestion definitely makes sense. Would you like to submit a PR since you already included the solution with the code, so you're almost there? :)

Thanks!

**Comment by MarkHollo** [comment]
Hi @peterfabian,

Out of all previously suggested accessibility enhancements, this is the only one that has not been resolved.

We wanted to submit a PR, but unfortunately, it returns a 403 forbidden access error so we're unable to help with this problem further.

I would just like to point out that the function in question has "moved" and is now on the following line https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/includes/wc-template-functions.php#L1225

Thanks and all the best!
